dev/com.ibm.ws.st.liberty.gradle/plugin.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# NLS_ENCODING=UNICODE
+# NLS_MESSAGEFORMAT_VAR
+
+pluginName=Gradle Build Integration Tools for Liberty
+IBMLibertyGradlePreferencePage=Liberty Integration
+
+navigatorContentProviderGradleRuntime=Liberty Gradle Runtime Environments
+popup.createRuntimeAndServer=Create WebSphere Liberty runtime and server
```

Comment:
Remove the WebSphere as it is just called Liberty server and Liberty runtime now.